docs/en/cloud/AWS-EMR.md

```diff
@@ -67,10 +51,8 @@ The `create-cluster` command requires passing in multiple flags to successfully
 The current version of Alluxio is compatible with `emr-5.25.0`.
 - `instance-count`: The number of nodes to provision for the cluster.
 - `instance-type`: The instance type to provision with.
-Make sure you pick an instance type supported by the Alluxio marketplace AMI.
 Note that your account is limited in the number of instances you can launch in each region;
 check your instance limits [here](https://console.aws.amazon.com/ec2/v2/home#Limits:).
-The default instance type for the AMI is `r4.4xlarge`.
```

Comment:
people should still pick this instance type by default. let's remove `for the AMI` to make sync easier

Reply:
done